src/Controller/SecurityController.php

```diff
@@ -7,11 +7,15 @@
 use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Component\Security\Http\Authentication\AuthenticationUtils;
 use App\Entity\User;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
 
 class SecurityController extends AbstractController
 {
+    // TODO: nie wiem, czy lepiej wyrzucić HttpException 403 jak tu, czy przekierować,
```

Comment:
@JanMaria Tak jak przy rejestracji, czyli zalogowany użytkownik zostaje przekierowany na stronę główną. 403 nie stosuje się w takim przypadku. To, że przekierowujemy to tylko dlatego, że nie ma żadnego sensu w tym, aby zalogowany użytkownik wchodził na stronę rejestracji/logowania.